### DISCUSSION TITLE
TOP 10 - Autarkie

### DISCUSSION THREAD

**Discussion by Zeppelin-PV** [solectrus/solectrus#3505] [Ideas]
die Autarkie ist mitunter das A&O einer PV
könnte man diese nicht auch noch bei den TOP 10 hinzufügen ?
![grafik](https://github.com/user-attachments/assets/7cd40595-85cb-49f9-a09c-1e7d747b0483)

**Comment by ledermann** [comment]
Ja, könnte man und wäre sicher auch interessant. Die Tücke steckt aber im Detail: Derzeitig stehen in der Top10 nur einfache, direkt gemessene Werte bereit. Der Grund ist, dass InfluxDB dies sehr schnell berechnen kann. Da die Autarkie aber nicht direkt gemessen wird, sondern sich aus anderen Werten ergibt, wäre das nur mit (völlig inakzeptablen) Performance-Einbußen möglich.

Möglicherweise ändern sich diese Rahmenbedingungen einmal, aber das sind momentan nur unausgereifte Idee. Insofern muss das dieser Vorschlag erst mal offen bleiben.
